src/config/internal.cpp

```diff
@@ -8,9 +9,37 @@
 
 #include <iterator>
 #include <optional>
+#include <oxen/log/format.hpp>
+
+using namespace oxen::log::literals;
 
 namespace session::config {
 
+namespace {
+
+    constexpr std::array<session::SessionIdPrefix, 6> all_session_id_prefixes = {
+            session::SessionIdPrefix::standard,
+            session::SessionIdPrefix::group,
+            session::SessionIdPrefix::community_blinded_legacy,
+            session::SessionIdPrefix::community_blinded,
+            session::SessionIdPrefix::version_blinded,
+            session::SessionIdPrefix::unblinded};
+
+    inline std::string to_string(session::SessionIdPrefix prefix) {
```

Comment:
```suggestion
    inline constexpr std::string_view to_string(session::SessionIdPrefix prefix) {
```
and then add `sv` to the end of all the strings (e.g. `return "00"sv).